src/diffpy/labpdfproc/tests/test_functions.py

```diff
@@ -80,3 +80,39 @@ def test_compute_cve(mocker):
         scat_quantity="cve",
     )
     assert actual_abdo == expected_abdo
+
+
+def test_apply_corr(mocker):
```

Comment:
this test is probably correct but hard to follow. 
1. there is a lot of duplicated code instantiating the DO's so maybe make a helper function called something like `_instantiate_test_do(yarray)` and just call that with only the y-array as input
2. maybe make the inputs easier to understand, so input y-array is [2., 2., 2.] and correction is [0.5, 0.5, 0.5] and so expected is a do with yarray [1., 1., 1.]. 

Would these make it easier to read?

If you make that wrapper function, on a separate PR we can maybe clean up some of the other tests as well.

Reply:
Sounds good, I'll do that